docs/design/README.md

```diff
@@ -150,12 +150,11 @@ import Console;
 fn Fibonacci(limit: i64) {
   var (a: i64, b: i64) = (0, 1);
   while (a < limit) {
-    Console.Print(a, " ");
+    Print("{0}", a);
     let next: i64 = a + b;
     a = b;
     b = next;
   }
-  Console.Print("\n");
```

Comment:
This example is supposed to demonstrate using an imported function. This is even more true in #2017 .

Reply:
How's this example with Math.Sqrt instead?